### PR TITLE
College data handler: Handles queries to the College Scorecard API

### PR DESCRIPTION
**Pull request by MikolJiron**
Query a list of colleges and render the list in a list view.

### CODE REVIEW

**Comment by MikolJiron** [comment]
@googlebot I consent.

**Comment by MikolJiron** [comment]
@googlebot I consent.

**Review by CarlosM10011 — APPROVED** [review]

**Comment by Lorraine-Lyu** [comment]
@googlebot I consent.

**Comment by MikolJiron** [comment]
@googlebot I consent.

File: src/main/webapp/js/datahandlers/collegelistdatahandler.js [hunk 1]
```diff
@@ -19,28 +19,56 @@
...
    */
-  async getNextBatch(batchIndex, itemsPerBatch) {
+  converter_(element) {
+    return {
+        name : element[NAME],
+        acceptance : element[ACCEPTANCE_RATE].toString(),
+        act : element[ACT_SCORE].toString(),
+        id : element[ID].toString()
+    };
+  }
```
Comment (Lorraine-Lyu) [inline src/main/webapp/js/datahandlers/collegelistdatahandler.js]: The code in this part is outdated. I use a list to represent each object, not sure if my changes will be approved. you can keep this code for now.

**Review by tinagrasso — APPROVED** [review]

**Review by hzimz — APPROVED** [review]

**Review by Lorraine-Lyu — APPROVED** [review]

**Pull request merged**